BSP/STM32F413/Makefile

```diff
@@ -77,7 +77,7 @@ SF = st-flash
 CPU = -mcpu=cortex-m4
 
 # float-abi
-FLOAT-ABI = -mfloat-abi=soft
+FLOAT-ABI = -mfloat-abi=hard
```

Comment:
has hardware float been proven to work on this branch? Also, is the default not hardware float, do we actually even need this flag anymore?

Reply:
hardware floats aren't still technically being used in the first place since we still use PedalToPercent in SendTritium anyways. but enabling this hasn't caused us any issues thus far.

Reply:
I think it is fine to merge it in as is because we don't use float. I do think this was part of the fix with correcting floats